core/include/scipp/core/element/comparison.h

```diff
@@ -8,6 +8,7 @@
 
 #include "scipp/common/overloaded.h"
 #include "scipp/core/element/arg_list.h"
+#include "scipp/core/element/math.h"
```

Comment:
Where is this needed?

Reply:
element wise `abs`

Reply:
I don't see that? You are using `abs` which should either be `std::abs` or the `abs` for `ValueAndVariance`?

Reply:
Yes you are right can be done directly. Should be fixed now.

Reply:
Ah, I see now with your update, it was using the lambda to avoid the direct call/lookup.